soc/nordic/common/vpr/soc_init.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <hal/nrf_vpr_csr.h>
+
+static int vpr_init(void)
+{
+	nrf_vpr_csr_rtperiph_enable_set(true);
```

Comment:
Add a comment here to explain why this is needed (will help to understand why this is here and not eg in a driver)

Reply:
Done